scripts/data/TransientState.js

```diff
@@ -145,6 +145,7 @@ export const saveMessage = async () => {
   }
   // Send the transient state to your API
   await fetch("http://localhost:8088/messages", postOptions)
+  window.alert(`Your message has been sent`)
```

Comment:
You can take off this window alert and attach `then(response => { 
    if (response.ok) {
      console.log("Post request successful!")
      window.alert("Your message has been sent")
    } else {
      console.log("Post request failed!")
      window.alert("Something went wrong")
    }
  })
  .catch(error => {
    console.error("An error occurred:", error);
  }) ` 
  directly after the 
  `await fetch("http://localhost:8088/messages", postOptions)`
  
  This will only send the success message once it is verified that the POST operation was successfully completed. It also will catch any errors that occur.